### ISSUE TITLE
Examples from the README have errors

### ISSUE THREAD

**Issue by sglindme** [convert-units/convert-units#298]
I'm on the 3.0.0 beta5 version

Using this script from the example: 

```import configureMeasurements from 'convert-units';

import length, {
  LengthSystems,
  LengthUnits,
} from "convert-units/definitions/length"

import area, {
  AreaSystems,
  AreaUnits,
} from "convert-units/definitions/area"

// Measures: The names of the measures being used
type Measures = 'length' | 'area';
// Systems: The systems being used across all measures
type Systems = LengthSystems | AreaSystems;
// Units: All the units across all measures and their systems
type Units = LengthUnits | AreaUnits;

const convert = configureMeasurements<Measures, Systems, Units>({
  length,
  area,
});

convert(4).from('m').to('cm');
// 400```

I get the errors:
Cannot find module 'convert-units/definitions/length' or its corresponding type declarations.ts(2307)
Cannot find module 'convert-units/definitions/area' or its corresponding type declarations.ts(2307)

**Comment by Taar** [comment]
I just published a new release `3.0.0-beta.6`, try that and let me know if you're still having issues.

**Comment by Taar** [comment]
Going to assume the latest version fixed this issue. Please open a new issue if that's not the case.
